gradle/libs.versions.toml

```diff
@@ -48,7 +49,7 @@ kotlin-reflect = { module = 'org.jetbrains.kotlin:kotlin-reflect' }
 
 tomcat-embed-core = { module = 'org.apache.tomcat.embed:tomcat-embed-core', version.ref = 'tomcat' }
 undertow-servlet = { module = 'io.undertow:undertow-servlet', version.ref = 'undertow' }
-jetty = { module = 'org.eclipse.jetty:jetty-servlet', version.ref = 'jetty' }
+managed-jetty = { module = 'org.eclipse.jetty:jetty-servlet', version.ref = 'managed-jetty' }
```

Comment:
Do we need to make this managed?

In the other linked PRs to this one, it's `jetty-server` not `jetty-servlet`

So could we just expose the version?

I don't know

Reply:
I think @timyates is right. @wetted change this PR to expose only the version.